lib/audio_common/.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
It seems like audio_common was not added correctly and the code is now in this repo and not the submodule. Maybe something broke when it was moved to `lib/`

Reply:
ok, checking.

Reply:
I not sure what happened. It seems work when I run `catkin build`.
What I have done for moving this `audio_commen`: create a new directory and move `audio_commen` inside, and then change the .gitmoudle file.

Reply:
Moving submodules is quite painful :/ I would recommend removing and readding it on a different location. What happened in here while moving the module:
- The module reference got removed from the file tree (but it is still in the `.gitmodules` file)
- The files of the submodule were no longer associated with the submodule and got added to this repo with the next commit.

This resulted in this PR adding 5k lines of code.

`catkin build` and the CI still pass as the code is still present in this repo, but it makes it hard to maintain as we have a copy instead of a reference.

Reply:
I will try to fix the issue and push the fixed version.

Reply:
It is fixed